spine-c/include/spine/Animation.h

```diff
@@ -91,8 +100,14 @@ typedef enum {
 
 struct spTimeline {
 	const spTimelineType type;
-
 	const void* const vtable;
+
+#ifdef __cplusplus
+	spTimeline() :
+		type(SP_TIMELINE_SCALE),
```

Comment:
Can be zero. All const struct fields are set by code in the `create` or `init` OO functions.